index.test.js

```diff
@@ -55,27 +56,28 @@ describe('handler', () => {
     process.env.SECONDS_THRESHOLD = 10
 
     s3.__addArrangement('itest-digest', {version:3, data: {t:'o', b: [100, 300]}})
-    decoder.__addBytes({ls: 'itest1', digest: 'itest-digest', start: 0, end: 198, time: 99998})
+    decoder.__addBytes({le: 'itest1', digest: 'itest-digest', start: 0, end: 198, time: 99998})
 
     const results1 = await handler()
-    expect(results1['itest1/itest-digest'].overall).toEqual(false)
-    expect(results1['itest1/itest-digest'].overallBytes).toEqual(99)
+    expect(results1['itest1/1970-01-01/itest-digest'].overall).toEqual(false)
+    expect(results1['itest1/1970-01-01/itest-digest'].overallBytes).toEqual(99)
     expect(kinesis.__records.length).toEqual(0)
 
     decoder.__clearBytes()
-    decoder.__addBytes({ls: 'itest1', digest: 'itest-digest', start: 199, end: 199, time: 99999})
+    decoder.__addBytes({le: 'itest1', digest: 'itest-digest', start: 199, end: 199, time: 99999})
 
     const results2 = await handler()
-    expect(results2['itest1/itest-digest'].overall).toEqual('seconds')
-    expect(results2['itest1/itest-digest'].overallBytes).toEqual(100)
+    expect(results2['itest1/1970-01-01/itest-digest'].overall).toEqual('seconds')
+    expect(results2['itest1/1970-01-01/itest-digest'].overallBytes).toEqual(100)
     expect(kinesis.__records.length).toEqual(1)
     expect(kinesis.__records[0]).toEqual({
-      time: 99999,
-      listenerSession: 'itest1',
+      type: 'bytes',
+      timestamp: 99999,
+      listenerEpisode: 'itest1',
       digest: 'itest-digest',
       bytes: 100,
       seconds: 10,
-      percent: 100 / 201
+      percent: 0.4975,
```

Comment:
yeah, i un-mocked a few things so this actually tests the records being put on kinesis.  rather than the params the `kinesis.js` lib gets called with.